apps/logo-quiz/src/main.tsx

```diff
@@ -8,34 +8,35 @@ import LogoVerify from './app/views/LogoVerify/LogoVerify';
 import LogoList from './app/views/LogoList/LogoList';
 import { Provider } from 'react-redux';
 import { applyMiddleware, createStore } from 'redux';
-import { rootReducer } from './store';
+import { rootReducer, logout } from './store';
 import thunkMiddleware from 'redux-thunk';
 import { createLogger } from 'redux-logger';
 import { composeWithDevTools } from 'redux-devtools-extension';
 import Login from './app/views/Login/Login';
+import './shared/api/http-interceptor';
 
 const loggerMiddleware = createLogger();
 
 const store = createStore(
   rootReducer,
-  composeWithDevTools(
-    applyMiddleware(
-      thunkMiddleware,
-      loggerMiddleware,
-    ),
-  ),
+  composeWithDevTools(applyMiddleware(thunkMiddleware, loggerMiddleware))
 );
 
+const onLogout = () => {
+  store.dispatch(logout());
+};
+
 const routing = (
   <Provider store={store}>
     <Router>
       <div>
-        <Route exact path="/" component={App}/>
-        <Route path="/logos/:id" component={LogoVerify}/>
-        <Route path="/levels/:id" component={LogoList}/>
-        <Route exact path="/levels" component={LevelList}/>
-        <Route exact path="/login" component={Login}/>
+        <Route exact path="/" component={App} />
+        <Route path="/logos/:id" component={LogoVerify} />
+        <Route path="/levels/:id" component={LogoList} />
+        <Route exact path="/levels" component={LevelList} />
```

Comment:
There is a typo in the import of this component (LOC: 6)

Reply:
Thanks for taking care of that!